### PR TITLE
Win32 test fixes

### PR DESCRIPTION
**Pull request by stinos**

### CODE REVIEW

File: tests/run-tests [hunk 1]
```diff
@@ -37,13 +37,18 @@ def run_micropython(pyb, args, test_file):
             with open(test_file, 'rb') as f:
                 line = f.readline()
                 if line.startswith(b'# cmdline:'):
-                    args += line[10:].strip().split()
+                    # subprocess.check_output on Windows only accepts strings, not bytes
+                    args += [c.decode('utf-8') for c in line[10:].strip().split()]
```
Comment (dpgeorge) [inline tests/run-tests]: Please use str(c, 'utf-8') instead of decode (it is more Python3 style).

File: tests/run-tests [hunk 2]
```diff
@@ -196,7 +201,7 @@ def run_tests(pyb, tests, args):
 
     # Some tests use unsupported features on Windows
     if os.name == 'nt':
-        skip_tests.add('import\\import_file.py') #works but CPython prints forward slashes
+        skip_tests.add('import/import_file.py') #works but CPython prints forward slashes
```
Comment (dpgeorge) [inline tests/run-tests]: Is the comment still relevant/needed with this change?

File: tests/run-tests (hunk 1 above)
Comment (stinos) [inline tests/run-tests]: ok fixed

File: tests/run-tests (hunk 2 above)
Comment (stinos) [inline tests/run-tests]: yes: using python, module.__file__ on windows prints path\to\file whereas uPy uses path/to/file

**Comment by dpgeorge** [comment]
Thanks, merged in 7ede3ec4b1ab4b77aac0dea0322a93b53f5df18a and dbfba6a20e252a2c8daa145099299283d98564de.

**Pull request closed without merging**